lib/util/hostname.js

```diff
@@ -0,0 +1,26 @@
+import assert from 'assert';
+import statelessDNSServer from 'stateless-dns-server';
+
+export default function getHostname(config, expires) {
+  let statelessConfig = config.statelessHostname || {};
+  if (!statelessConfig || !statelessConfig.enabled) {
+    return config.host;
+  }
+
+  let secret = statelessConfig.secret || '1234';
+  let domain = statelessConfig.domain || '';
+  assert(secret, 'Must supply a secret for stateless dns server');
+  assert(domain, 'Must supply a domain name used for stateless hostname');
+
+  let hostname;
+  let ip = config.publicIp || '127.0.0.1';
+  ip = ip.split('.').map((octet) => { return parseInt(octet) });
```

Comment:
Okay, so just because we were discussing how to do this best, I since found that, this works too:
```js
ip = ip.split('.').map(digit => parseInt(digit));
```
He he, just as an FYI... I don't care how do it here...